packages/twenty-server/src/modules/opportunity/standard-objects/opportunity.object-metadata.ts

```diff
@@ -102,18 +101,6 @@ export class OpportunityObjectMetadata extends BaseObjectMetadata {
   @IsNullable()
   position: number;
 
-  // Relations
```

Comment:
I'm wondering if we should invalidate schema cache after this kind of update? Especially now that it is in Redis instead of in-memory. Didn't we have issues in the past?

Reply:
it is invalidated when we sync metadata!